backend/src/test/java/quarano/actions/DiaryEventListenerTest.java

```diff
@@ -325,25 +338,193 @@ void testOnDiaryEntryMissing() {
 		assertThat(itemCaptor.getValue().getDescription().getCode()).isEqualTo(DescriptionCode.DIARY_ENTRY_MISSING);
 	}
 
-	@Test
-	void testNoActionItemForIndexPerson() {
+	@Test // CORE-593
+	void testDifferentTemperatureThresholdsForIndexAndContact() {
 
-		var now = Slot.now();
+		// for contact
 		var person = TrackedPersonIdentifier.of(UUID.randomUUID());
-		var entry = mock(DiaryEntry.class);
-		var event = DiaryEntry.DiaryEntryAdded.of(entry);
-		var trackedCase = mock(TrackedCase.class);
+		var entry = DiaryEntry.of(Slot.now(), person)
+				.setBodyTemperature(BodyTemperature.of(41F));
 
-		when(entry.getTrackedPersonId()).thenReturn(person);
-		lenient().when(entry.getSlot()).thenReturn(now);
-		lenient().when(entry.getBodyTemperature()).thenReturn(BodyTemperature.of(42.0f)); // body temp well above conf threshold
-		when(cases.findByTrackedPerson(person)).thenReturn(Optional.of(trackedCase));
-		when(trackedCase.isIndexCase()).thenReturn(true);
-		when(items.findDiaryEntryMissingActionItemsFor(person, now)).thenReturn(ActionItems.empty());
+		when(diaryManagement.findDiaryFor(person)).thenReturn(Diary.of(Streamable.empty()));
+		when(items.findUnresolvedByDescriptionCode(person, DescriptionCode.INCREASED_TEMPERATURE))
+				.thenReturn(ActionItems.empty());
+		when(cases.findTypeByTrackedPerson(person)).thenReturn(Optional.of(CaseType.CONTACT));
 
-		listener.on(event);
+		listener.handleDiaryEntryForBodyTemperature(entry);
 
-		verify(items, times(0)).save(itemCaptor.capture());
+		verify(items, times(1)).save(itemCaptor.capture());
+
+		var item = itemCaptor.getValue();
+
+		assertThat(item.getType()).isEqualTo(ActionItem.ItemType.MEDICAL_INCIDENT);
+
+		var description = item.getDescription();
+		assertThat(description.getArguments()).isEqualTo(new Object[] { "41,0°C", "41,0°C" });
+		assertThat(description.getCode()).isEqualTo(DescriptionCode.INCREASED_TEMPERATURE);
+
+		// for index
+		entry = DiaryEntry.of(Slot.now(), person)
+				.setBodyTemperature(BodyTemperature.of(43F));
+
+		when(cases.findTypeByTrackedPerson(person)).thenReturn(Optional.of(CaseType.INDEX));
+
+		listener.handleDiaryEntryForBodyTemperature(entry);
+
+		verify(items, times(1 + 1)).save(itemCaptor.capture());
+
+		item = itemCaptor.getValue();
+
+		assertThat(item.getType()).isEqualTo(ActionItem.ItemType.MEDICAL_INCIDENT);
+
+		description = item.getDescription();
+		assertThat(description.getArguments()).isEqualTo(new Object[] { "43,0°C", "42,0°C" });
+		assertThat(description.getCode()).isEqualTo(DescriptionCode.INCREASED_TEMPERATURE);
+	}
+
+	@Test // CORE-593
+	void testOnDiaryEntryAddedWithConspicuousSymptoms() {
+
+		var person = TrackedPersonIdentifier.of(UUID.randomUUID());
+		var symptom = mock(Symptom.class);
+		var entry = DiaryEntry.of(Slot.now(), person)
+				.setSymptoms(List.of(symptom))
+				.setBodyTemperature(BodyTemperature.of(36));
+
+		when(symptom.isConspicuousAtContact()).thenReturn(true);
```

Comment:
Does it make sense to simply load the `Symptom` rather than setting up some elaborate mocking?

Reply:
I don't think it makes sense to turn the unit test into an integration test so we can load symptoms. Or I'm misunderstanding you. I did put the symptom mocking into a method, though.